custom_components/tuya_local/devices/wistar_roller_blind_nopos.yaml

```diff
@@ -138,3 +146,8 @@ secondary_entities:
       - id: 103
         type: boolean
         name: switch
+        mapping:
```

Comment:
I don't think it is necessary to flip the meaning of the tuya dp, as this no longer has a special mapping within the cover entity, so it does not affect the behaviour of the cover like it used to.

Reply:
In my case Tuya reports the meaning of Reversed opposite of how you'd display it and makes it appear correctly.